amethyst_audio/src/output.rs

```diff
@@ -75,11 +67,17 @@ impl Output {
     ) -> Result<(), DecoderError> {
```

Comment:
Shouldn't this be 3 functions instead of 2?
play // actual play logic. takes the stuff you might reuse as parameter
play_once // obvious
play_multiple // calls play n times

Its weird design wise to have play_n_times(1) and it also makes you create a int for nothing (for index).

Also, play_multiple VS play_n_times naming.

Reply:
That structure doesn't work here because just calling play multiple times won't get the desired result.  play_n_times doesn't play all the sounds at once, it queues them into a sink which will then play them sequentially.  I don't understand what your distinction would be between play and play_once though.

ints on the stack are super duper cheap, so calling it with 1 as the parameter is just a way to reduce code duplication, mainly so that changes done in one place aren't reflected in another.

Reply:
Sorry disregard the first paragraph there, it just clicked what you mean.

Reply:
I'm not quite sure if I favor that approach because it requires me to add another function and interface for little gain imo.

Reply:
it'd be private